diff --git a/drivers/bluetooth/Kconfig b/drivers/bluetooth/Kconfig
index a617ecf93b0d..6996f2a9f326 100755
--- a/drivers/bluetooth/Kconfig
+++ b/drivers/bluetooth/Kconfig
@@ -5,7 +5,6 @@ menu "Bluetooth device drivers"
 config BT_HCIBTUSB
 	tristate "HCI USB driver"
 	depends on USB
-	select BT_ATH3K
 	help
 	  Bluetooth HCI USB driver.
 	  This driver is required if you want to use Bluetooth devices with
diff --git a/drivers/bluetooth/btusb.c b/drivers/bluetooth/btusb.c
index f8beda629982..97def029fed3 100644
--- a/drivers/bluetooth/btusb.c
+++ b/drivers/bluetooth/btusb.c
@@ -37,8 +37,8 @@ static bool ignore_sniffer;
 static bool disable_scofix;
 static bool force_scofix;
 
-static bool sco_conn;
-static bool reset = 1;
+static int sco_conn;
+static int reset = 1;
 
 static struct usb_driver btusb_driver;
 
diff --git a/drivers/usb/phy/class-dual-role.c b/drivers/usb/phy/class-dual-role.c
index 51fcb545a9d5..ce889dd529cb 100644
--- a/drivers/usb/phy/class-dual-role.c
+++ b/drivers/usb/phy/class-dual-role.c
@@ -86,7 +86,7 @@ void dual_role_instance_changed(struct dual_role_phy_instance *dual_role)
 	pm_wakeup_event(&dual_role->dev, DUAL_ROLE_NOTIFICATION_TIMEOUT);
 	schedule_work(&dual_role->changed_work);
 }
-EXPORT_SYMBOL_GPL(dual_role_instance_changed);
+EXPORT_SYMBOL_GPL(dual_role_instance_changed)
 
 int dual_role_get_property(struct dual_role_phy_instance *dual_role,
 			   enum dual_role_property prop,
